src/pages/index.js

```diff
@@ -72,7 +78,7 @@ const newCardFormValidator = new FormValidator(options, popupCardForm);
 newCardFormValidator.enableValidation();
 
 function createCard(cardData) {
-  const card = new Card(cardData, '.template-card', handlePicturePopup);
+  const card = new Card(cardData, '.template-card', handlePicturePopup, handleDeleteButton, handleLikeButton);
```

Comment:
Could be better: you can pass the arguments as an object, so it won't be possible to make mistakes in Card class constructor